tools/generate_deb.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+DF_VERSION="0.12.0"
+
+
+                echo "Building deb package for dragonflydb $DF_VERSION" 
+                echo "Generating files"
+                mkdir -p dragonfly_"$DF_VERSION"_amd64/usr/bin/
+                mkdir -p dragonfly_"$DF_VERSION"_amd64/etc/systemd/system
```

Comment:
There are lots of repetition in the code. Why not introduce a variable ROOT=dragonfly_"$DF_VERSION"_amd64 and use it everywhere?